src/extension.ts

```diff
@@ -1510,12 +1446,12 @@ export class ExtensionManager implements vscode.Disposable {
             return false;
         }
 
```

Comment:
For this and the next few presets related functions, we need to determine whether the approach is still valid or if we should be passing in a CMakeProject as input instead of a workspace folder.

Reply:
I need to investigate this everywhere we used `getProjectsForWorkspaceFolder`